spikeinterface/toolkit/postprocessing/principal_component.py

```diff
@@ -64,6 +64,23 @@ def _set_params(self, n_components=5, mode='by_channel_local',
                       dtype=np.dtype(dtype).str)
         
         return params
+    
+    def _filter_units(self, unit_ids, new_waveforms_folder):
+        # creaste new extension folder
+        new_pca_folder = new_waveforms_folder / self.extension_name
+        new_pca_folder.mkdir()
+        # copy parameter file
+        shutil.copyfile(self.extension_folder / "params.json",
+                        new_pca_folder / "params.json")
+
```

Comment:
This is generic and must be shared in base in `filter_units(self)`

Reply:
done